iotdb/src/main/java/org/apache/iotdb/db/engine/memcontrol/RecordMemController.java

```diff
@@ -154,6 +139,7 @@ public void releaseUsage(Object user, long freeSize) {
           .error("{} requests to free {} bytes while it only registered {} bytes", user,
               freeSize, usage);
       totalMemUsed.addAndGet(-usageLong);
+      usage.set(0);
```

Comment:
don't you remove it from memMap?

Reply:
Because it does not indicate that the user will not use more memory when its usage drop to 0.